cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/AppFabricTestBase.java

```diff
@@ -154,6 +157,8 @@ public static void beforeClass() throws Throwable {
     metricsService.startAndWait();
     streamService = injector.getInstance(StreamService.class);
     streamService.startAndWait();
+    serviceStore = injector.getInstance(ServiceStore.class);
```

Comment:
We did not have MonitorHandler tests before. So needed this so that the servicestore is initialized correctly.